README.md

```diff
@@ -31,11 +31,16 @@ For more information about minikube, see the [proposal](https://github.com/kuber
 * Simplifying kubernetes production deployment experience. Kube-deploy is attempting to tackle this problem.
 * Supporting all possible deployment configurations of Kubernetes like various types of storage, networking, etc.
 
+## Build Requirements
+
+* A recent Go distribution (>1.5)
```

Comment:
Done. We can probably drop the: export GO15VENDOREXPERIMENT=1
from our Makefile eventually.